---
event_type: IssueCommentEvent
avatar: "https://avatars.githubusercontent.com/u/741970?"
user: grondo
date: 2023-08-03
repo_name: conda-forge/flux-core-feedstock
html_url: https://github.com/conda-forge/flux-core-feedstock/pull/23
repo_url: https://github.com/conda-forge/flux-core-feedstock
---

<a href='https://github.com/grondo' target='_blank'>grondo</a> commented on issue <a href='https://github.com/conda-forge/flux-core-feedstock/pull/23' target='_blank'>conda-forge/flux-core-feedstock#23</a>.

<small>No it doesn't fail for us in CI.. also nothing has changed in flux-core related to that code in this release (relative to 0.52.0) so I'm not sure why it would be failing. Do you have the output of the failing test perchance?...</small>

<a href='https://github.com/conda-forge/flux-core-feedstock/pull/23' target='_blank'>View Comment</a>